CMakeLists.txt

```diff
@@ -202,25 +213,46 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
         find_package(KokkosKernels REQUIRED)
     endif()
 
-    target_link_libraries(DDC INTERFACE Kokkos::kokkoskernels)
-    target_compile_definitions(DDC INTERFACE "DDC_BUILD_KERNELS_SPLINES")
+    add_library(ddc_splines INTERFACE)
+    add_library(DDC::splines ALIAS ddc_splines)
+    install(FILES cmake/FindLAPACKE.cmake DESTINATION lib/cmake/DDC)
+    install(TARGETS ddc_splines EXPORT DDCTargets)
+    target_compile_definitions(ddc_splines INTERFACE "DDC_BUILD_KERNELS_SPLINES")
+    target_include_directories(ddc_splines SYSTEM INTERFACE ${LAPACKE_INCLUDE_DIRS})
+    target_link_libraries(
+        ddc_splines
+        INTERFACE ddc_core Ginkgo::ginkgo Kokkos::kokkos Kokkos::kokkoskernels ${LAPACKE_LIBRARIES}
+    )
+
+    ### DDC legacy target
+    target_link_libraries(DDC INTERFACE ddc_splines)
 endif()
 
 ## The PDI wrapper
 
 if("${DDC_BUILD_PDI_WRAPPER}")
-    add_library(PDI_Wrapper INTERFACE)
-    target_compile_features(PDI_Wrapper INTERFACE cxx_std_17)
+    add_library(ddc_pdi INTERFACE)
+    add_library(DDC::pdi ALIAS ddc_pdi)
+    install(TARGETS ddc_pdi EXPORT DDCTargets)
+    target_compile_definitions(ddc_pdi INTERFACE "DDC_BUILD_PDI_WRAPPER")
+    target_compile_features(ddc_pdi INTERFACE cxx_std_17)
     target_include_directories(
-        PDI_Wrapper
+        ddc_pdi
         INTERFACE
             "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
             "$<INSTALL_INTERFACE:include>"
     )
-    target_link_libraries(PDI_Wrapper INTERFACE DDC::DDC PDI::PDI_C)
-    target_compile_definitions(PDI_Wrapper INTERFACE "DDC_BUILD_PDI_WRAPPER")
+    target_link_libraries(ddc_pdi INTERFACE ddc_core PDI::PDI_C)
+
+    ### PDI_Wrapper legacy target
+    add_library(PDI_Wrapper INTERFACE)
+    set_target_properties(PDI_Wrapper PROPERTIES DEPRECATION "Use DDC::pdi instead")
     add_library(DDC::PDI_Wrapper ALIAS PDI_Wrapper)
     install(TARGETS PDI_Wrapper EXPORT DDCTargets)
+    target_link_libraries(PDI_Wrapper INTERFACE ddc_pdi)
```

Comment:
Why not just an ALIAS?

Reply:
Do you think it works well with the DEPRECATION property ? I would expect that `DDC::pdi` to become also deprecated ?

Reply:
Good question. I've never used the `DEPRECATION` property

Reply:
You're right, properties can not be set on ALIASes. I guess an interface is the way to go 👍